utils/sql/git/required/2021_02_14_npc_exp_mod.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `npc_types` ADD COLUMN `exp_mod` INT NOT NULL DEFAULT '100' AFTER `always_aggro`;
```

Comment:
I think for the database type SMALLINT unsigned would be sufficient (0 to 65535), that would be at most 655times the normal EXP. Depending on the amount of EXP, such an increase could otherwise also lead to overflow in "uint32 Client::GetExperienceForKill(Mob *against)".